src/cts/src/TreeBuilder.h

```diff
@@ -73,6 +73,12 @@ struct pointEqual
   }
 };
 
+using TreeType = enum TreeType {
```

Comment:
"enum TreeType {" suffices - its already in scope so there is no need for a 'using'.

It is generally advised to use "enum class" rather than old style enums unless they are already in a limited scope (ie inside a class).

Reply:
This was a clang-tidy suggestion, so I will keep this.

Reply:
I tried removing it locally and didn't get any clang-tidy message.  What did you see?

Reply:
Oh, this may be because I was using typedef initially.    I changed it per your suggestion now.